apps/static-site/src/components/iiif/DownloadImage.tsx

```diff
@@ -59,7 +76,13 @@ function DownloadImageInner({ single, content }: { single?: boolean; content: {
             <li key={largest.width} className="flex items-center gap-2">
               <DownloadIcon className="text-2xl text-slate-300 opacity-50" />
               <a className="underline" href={`${service.id}/full/${largest.width},/0/default.jpg`}>
-                {content.download} ({largest.width} x {largest.height})
+                {content.downloadImage} ({largest.width} x {largest.height})
+              </a>
+            </li>
+            <li key={width} className="flex items-center gap-2">
+              <DownloadIcon className="text-2xl text-slate-300 opacity-50" />
+              <a className="underline" href={`${service.id}/full/max/0/default.jpg`}>
```

Comment:
The DLCS should respect the `max` property 👍

Reply:
Yes it does, but this image:
https://dlc.services/iiif-img/v3/7/6/9d561e21-51be-459b-b8c7-2191dc8dd382/info.json

Has a maxArea of 10000000 pixels.

If you request the full/max image:
https://dlc.services/iiif-img/v3/7/6/9d561e21-51be-459b-b8c7-2191dc8dd382/full/max/0/default.jpg

It returns the full image of 4968 x 3696 pixels (18361728 pixels).

So this is why I did not use the added function to adjust the dimensions.